test/library/packages/UnitTest/PopulateTests_PRIM/TestCount.compopts

```diff
@@ -0,0 +1 @@
+--legacy-nilable-classes
```

Comment:
For other UnitTest tests, I've been adding `borrowed Test` to the arguments of the tests. That is, assuming you are seeing an error about unable to make a generic FCF.

Reply:
PRIM_GATHER_TESTS does not work with --no-legacy-nilable-classes due to #13720.